app/controllers/gcs_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'google/cloud/storage'
+
+class GcsController < ApplicationController
+
+  # POST /gcs
+  def create
+
+    bucket_name = 'scl-lny-receipts'
+    file_name = gcs_params[:file_name]
+    file_type = gcs_params[:file_type]
+
+    storage = Google::Cloud::Storage.new
+    storage_expiry_time = 15 * 60 # 5 minutes
```

Comment:
i think this is actually 15 minutes but shouldn't matter too much. if someone has the client open for a long period of time they might get an error but i dont know if sessions will last > 15 minutes

Reply:
Good catch, I forgot to update my comment. The UI flow has the getting signed URL and upload back-to-back, so these should in theory never be far apart, I think we could lower this back to 5, but I'll do some testing.